starter/docs/commands/root_completion_bash.md

```diff
@@ -4,7 +4,7 @@ Generate command completion script for Bash shell
 
 ### Synopsis
 
-' Generate command completion script for Bash shell. Installing on Linux: 1. Run `rhoas completion bash > rhoas_completions` to create the script file. 2. Run `sudo mv rhoas_completions /etc/bash_completion.d/rhoas` to place the script in a special Bash completions directory. Installing on macOS: 1. Run `rhoas completion bash > rhoas_completions` to create the script file. 2. Run `sudo mv rhoas_completions /usr/local/etc/bash_completion.d/rhoas` to place the script in a special Bash completions directory. '
+' Generate command completion script for Bash shell. Installing on Linux: 1. Run `<no value> completion bash > <no value>_completions` to create the script file. 2. Run `sudo mv <no value>_completions /etc/bash_completion.d/<no value>` to place the script in a special Bash completions directory. Installing on macOS: 1. Run `<no value> completion bash > <no value>_completions` to create the script file. 2. Run `sudo mv <no value>_completions /usr/local/etc/bash_completion.d/<no value>` to place the script in a special Bash completions directory. '
```

Comment:
I think the `{{ .CliName }}` placeholder is being replaced by `<no value>` values over here, since `CliName` var is empty currently.
So maybe you can add a default value to it, to fix it temporarily.

WDYT?

Reply:
thanks, I didn't noticed this. Will fix it.